### PR TITLE
Condition Example

### PR DESCRIPTION
**Pull request by FinalT**
Condition Example

### CODE REVIEW

**Comment by justxuewei** [comment]
Is there any way to use conditional route without config center or dubbo admin? Because doing that is a very heavy burden to the users who are new to the Dubbo-go.

**Comment by FinalT** [comment]
> Is there any way to use conditional route without config center or dubbo admin? Because doing that is a very heavy burden to the users who are new to the Dubbo-go.
I think there is a way to do it, and I will do it

**Comment by FinalT** [comment]
> Is there any way to use conditional route without config center or dubbo admin? Because doing that is a very heavy burden to the users who are new to the Dubbo-go.

I think there is a way to do it, and I will do it

File: condition/README_zh.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# dubbo-go condition 示例
+
+可以按照下文步骤，将 Demo 部署到本地。
+
+<h2 id="target">1 条件路由介绍</h2>
```
[the remark this answers is not in the document]
Comment (FinalT) [reply condition/README_zh.md:5]: I will revise it

**Comment by FinalT** [comment]
> Is there any way to use conditional route without config center or dubbo admin? Because doing that is a very heavy burden to the users who are new to the Dubbo-go.

Now there is a [problem](https://github.com/apache/dubbo-go/issues/2283), I don't have a good way to get the routing rules locally

**Comment by justxuewei** [comment]
> Now there is a [problem](https://github.com/apache/dubbo-go/issues/2283), I don't have a good way to get the routing rules locally

I am just wondering, do the conditional rules only get from the config center, or can they also be able to set in the config file?

**Comment by FinalT** [comment]
> 



> Is there any way to use conditional route without config center or dubbo admin? Because doing that is a very heavy burden to the users who are new to the Dubbo-go.



> > Now there is a [problem](https://github.com/apache/dubbo-go/issues/2283), I don't have a good way to get the routing rules locally
> 
> I am just wondering, do the conditional rules only get from the config center, or can they also be able to set in the config file?

Yes，Can be able to set in the config file

**Comment by justxuewei** [comment]
> Yes，Can be able to set in the config file

Okay, could you provide an example of how to use the feature locally, as a brief guide to help users understand how it works? It's not necessary to use it with the Dubbo admin. If you believe that obtaining the rules dynamically is also an important topic, then leaving both samples would be fine with me.

**Comment by FinalT** [comment]
> > Yes，Can be able to set in the config file
> 
> Okay, could you provide an example of how to use the feature locally, as a brief guide to help users understand how it works? It's not necessary to use it with the Dubbo admin. If you believe that obtaining the rules dynamically is also an important topic, then leaving both samples would be fine with me.
> 
> To summarize, keeping your examples as simple as you could.

Ok, I'll finish it.

**Pull request closed without merging**